.eslintrc.js

```diff
@@ -0,0 +1,30 @@
+// http://eslint.org/docs/user-guide/configuring
```

Comment:
Why not keep `.eslintrc.json` instead of `.eslintrc.js`? I'm not 100% sure third-party tools like Phpstorm/Webstorm and Codacy work with that extension.

Reply:
It's really easier to configure with a js file.

from PhpStorm:
```
With Automatic search, PhpStorm looks for a .eslintrc file or tries to detect a configuration defined under eslintConfig in a package.json. PhpStorm first looks for a .eslintrc or package.json in the folder with the file to be linted, then in its parent folder, and so on up to the project root. If the search fails, ESLint uses its default embedded configuration file.
Choose Configuration File to use a custom file and specify the file location in the Path field.
```

Eslint take `.eslintrc.*` and codacy do the same :)

Reply:
From eslint configuration:
```
The first way to use configuration files is via .eslintrc.* and package.json files. ESLint will automatically look for them in the directory of the file to be linted, and in successive parent directories all the way up to the root directory of the filesystem (unless root: true is specified). This option is useful when you want different configurations for different parts of a project or when you want others to be able to use ESLint directly without needing to remember to pass in the configuration file.

```